examples/gallery/line/vector-heads-tails.py

```diff
@@ -0,0 +1,44 @@
+"""
+Vector heads and tails
+-------
+
+The :meth:`pygmt.Figure.plot` method can plot vectors with individual 
+heads and tails. We must specify the modifiers (together with the vector type)
+by passing the corresponding shortcuts to the  ``style`` argument. 
+
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+fig.basemap(
+    region=[0, 10, 0, 15], projection="X15c/10c", frame='+t"Vector heads and tails"'
+)
+
+x = 1
+y = 14
+angle = 0  # in degrees, measured counter-clockwise from horizontal
+length = 7
+
+for vecstyle in [
+    "v0c",  # vector without head and tail (line)
```

Comment:
I think `v0c` is equivalent to `v`.

Reply:
Thought the same but it isn't. Only `v` gives me nothing back.